test/cli.test.js

```diff
@@ -9,15 +9,30 @@ var vtshave_cli = path.resolve(__dirname, '..', 'bin', 'vtshave.js');
 var vtshaver_filters_cli = path.resolve(__dirname, '..', 'bin', 'vtshaver-filters.js');
 var tile = path.join(__dirname, 'fixtures', 'tiles', 'sf_16_10465_25329.vector.pbf');
 var style = path.join(__dirname, 'fixtures', 'styles', 'bright-v9.json');
+var styleWithoutLegacyExpressions = path.join(__dirname, 'fixtures', 'styles', 'expressions.json');
 
 if (process.env.TOOLSET && process.env.TOOLSET === 'asan') {
     test('vtshave cli works - SKIPPED due to ASAN build', function(t) { t.end() });
 } else {
     test('vtshave cli works', function(t) {
-      var args = [vtshave_cli, '--tile', tile, '--style', style, '--zoom', 16];
-      spawn(process.execPath, args)
+      var args = [vtshave_cli, '--tile', tile, '--style', styleWithoutLegacyExpressions, '--zoom', 16];
+      const child = spawn(process.execPath, args);
+
+      var logs = '';
+
+      child.stdout.setEncoding('utf8');
+      child.stdout.on('data', function(chunk) {
+          logs += chunk;
+      });
+
+      child.stderr.on('data', function(chunk) {
+          logs += chunk;
+      });
+
+      child
           .on('error', function(err) { t.ifError(err, 'no error'); })
           .on('close', function(code) {
+              console.log(logs);
```

Comment:
@lukaswelte If this is just for one-time debugging this should be removed. If you can think of a good way to turn this into an assert, that would be ideal.

Reply:
Agree, now they are only printed in the case of the assertion error. I can't directly think of a better way. If that is no good then let me know and I'll remove it

Reply:
This seems good, thanks